cmd/cbnt-prov/cmd.go

```diff
@@ -927,19 +944,59 @@ func (p printFITCmd) Run(ctx *context) error {
 	return nil
 }
 
+func (v *verifyKMSigCmd) Run(ctx *context) error {
+	kmRaw, err := ioutil.ReadFile(v.KM)
+	if err != nil {
+		return err
+	}
+
+	var km key.Manifest
+	r := bytes.NewReader(kmRaw)
+	_, err = km.ReadFrom(r)
+	if err != nil {
+		return err
+	}
+	if err := km.KeyAndSignature.Verify(kmRaw[:km.KeyAndSignatureOffset()]); err != nil {
```

Comment:
Why are we doing the "one-liner" here and above explicitly error checking?

Reply:
done